protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2047,9 +2048,9 @@ message ListMarginLevelsResponse {
 // If a party ID is provided, the stream will contain margin levels for that party only.
 // Optionally, the list can be additionally filtered by market
 message ObserveMarginLevelsRequest {
-  // Party ID to filter for. If empty, no parties will be filtered.
+  // Restrict margin level updates to those relating to the given party.
   string party_id = 1;
-  // Optional Market ID to filter for. If omitted, no markets will be filtered.
+  // Restrict margin level updates to those relating to the given market.
```

Comment:
Some include the "If not set..." caveat, and some don't. Is that a conscious choice for those where it's more obvious what the alternative is?

Reply:
Yea thats what I was trying to go with. So with all the filtering/restricting ones (to me at least) you would expect not setting anything would give you everything.

But there are other optional fields where its not that obvious, like when you can specific an historic epoch. Us returning the latest epoch is a choice we made, it could also make sense to return the first one. So for those I feel it makes sense to add a "If not set" bit.